Tests/tests_e2e/content/xsoar_saas/test_e2e_xsoar_saas.py

```diff
@@ -133,9 +134,15 @@ def test_taxii2_server_returns_indicators(
                     indicators = get_json_response(response).get("objects")
                     assert indicators, f'could not get indicators from url={response.request.url} with available ' \
                         f'indicators={available_indicators}, status code={response.status_code}, response={indicators}'
-        except ApiException as error:
-            logging.info(f'Got error when running test_taxii2_server_returns_indicators with {port=} error:\n{error}')
-            if port == "8002":
+        except (ApiException, RequestException) as error:
+            if isinstance(error, ApiException):
+                logging.error(f'Got error when running test_taxii2_server_returns_indicators with {port=}, error:\n{error}')
+            else:
+                logging.error(
+                    f'Got error response {error.response} when running '
+                    f'test_taxii2_server_returns_indicators with {port=} when sending request {error.request}'
+                )
+            if port == "8004":
```

Comment:
better to flip a boolean in the loop and raise outside the loop if it wasnt flipped

Reply:
what's the value in that?